internal/ui/app.go

```diff
@@ -199,6 +201,23 @@ func (a *App) quitCmd(evt *tcell.EventKey) *tcell.EventKey {
 	return nil
 }
 
+func (a *App) suspendCmd(evt *tcell.EventKey) *tcell.EventKey {
+	if a.InCmdMode() {
+		return evt
+	}
+
+	if !a.Config.K9s.NoSuspendOnCtrlZ {
```

Comment:
We would not be needed the check here if we don't enable the action in the first place?

Reply:
removed ;)